### ISSUE TITLE
Improvements to "Whitelist a user" admin page.

### ISSUE THREAD

**Issue by jpmedley** [GoogleChrome/chromium-dashboard#675]
We should probably have several things, though I realize these have different levels of complexity.

* __Display all emails in the list:__ currently the list is longer than can be scrolled to. I have no way of determining if a name is already present.
* __Display the last time a user updated a page:__ We probably have names that need to be removed, but no way to determine that. 

@robdobson Maybe our 20% volunteer could work on this?

**Comment by robdodson** [comment]
I think our 20%'er is still heads down on migrating us off of v0 web components. I actually need to check in with him to see how things are going...

**Comment by jpmedley** [comment]
Makes sense. I was just trying to suggest a way we might prioritize it. This is a concern to me since I'm often approached to add emails. We have more non-googlers than ever contributing to Chromium. It would be nice to have better controls over who has dashboard access.
